Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.8-slim
+WORKDIR /mote
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+RUN sed -i '/use_memcached = / s/False/True/' mote/config.py
```

Comment:
Please remove this line, as it asks the config to set `use_memcached` as True from False (By default). And as we're using file based cache, we won't be needing this line.

Reply:
Yes, I was about to say that for we made those changes via https://github.com/fedora-infra/mote/pull/150/commits/46dc6fb7b4f125117cc26ec9abe9b29dcaa7f40c.